src/app/features/courses/components/course-list/course-list.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { CourseCard } from '../course-card/course-card.component';
+
+export type CourseBtnAction = 'show' | 'edit' | 'remove';
+
+@Component({
+  selector: 'app-course-list',
+  templateUrl: './course-list.component.html',
+  styleUrls: ['./course-list.component.scss']
+})
+export class CourseListComponent implements OnInit {
+  @Input() courseList: CourseCard[] = [];
+  @Input() isEditable: boolean = false;
```

Comment:
could be just `@Input() isEditable = false;` and type boolean will be implicitly assigned